chemprop/nn/metrics.py

```diff
@@ -43,22 +43,27 @@
 
 
 class Metric(LossFunction):
+    """
+    .. note::
+        This class inherits from :class:`LossFunction`, which requires task_weights. These
+        task_weights are not used in Metric and can be set to any Tensor.
```

Comment:
Probably better to just copy the docstring from `LossFunction` here and make the appropriate changes. That way a user doesn't need to cross-reference the `LossFunction` docstring to learn about a `Metric`.